### PR TITLE
bump kubekins tag from config/jobs as well

### PR DESCRIPTION
**Pull request by krzyzacy**
/area jobs
/assign @BenTheElder

### CODE REVIEW

File: experiment/bump_e2e_image.sh [hunk 1]
```diff
@@ -36,44 +36,40 @@ if [[ -n ${dirty} ]]; then
...
+  "--yaml-config-path=${TREE}/experiment/test_config.yaml" \
+  "--json-config-path=${TREE}/jobs/config.json" \
+  "--prow-config-path=${TREE}/prow/config.yaml"
 bazel run //jobs:config_sort
-popd
 
 # Scan for kubekins-e2e:v.* as a rudimentary way to avoid
 # replacing :latest.
 $SED -i "s/\/kubekins-e2e:v.*-\(.*\)$/\/kubekins-e2e:${TAG}-\1/" "${TREE}/prow/config.yaml"
+find "${TREE}/config/jobs/" -type f -name \*.yaml -exec $SED -i "s/\/kubekins-e2e:v.*-\(.*\)$/\/kubekins-e2e:${TAG}-\1/" {} \;
```
Comment (BenTheElder) [inline experiment/bump_e2e_image.sh]: these \ should be \\
https://github.com/kubernetes/test-infra/pull/8032/files#diff-2649426475dffa0e0587adf3e8822a33

Comment (krzyzacy) [reply experiment/bump_e2e_image.sh]: ok I took #8032 and added that one line :-)

**Comment by BenTheElder** [comment]
```sh
TAG="${DATE}-$(git describe --tags --always --dirty)"
pushd "${TREE}/images/kubeadm"
make push TAG="${TAG}"
popd
```
can we fix that? why are we re-computing the tag? and we can still drop all the push/popd and just reference things from ${TREE}?

/cc @rmmh this will also conflict with the shellcheck fixes but fixes some other things

**Comment by krzyzacy** [comment]
gahhh lol I thought it was fixed on the other PR as well

**Comment by BenTheElder** [comment]
ah nope, those are valid bash but in this case they just add unnecessary
state to follow

**Comment by krzyzacy** [comment]
done here

**Pull request merged**